reactor-test/src/main/java/reactor/test/StepVerifier.java

```diff
@@ -322,6 +322,25 @@ static void resetDefaultTimeout() {
 	 */
 	Assertions verifyThenAssertThat();
 
+	/**
+	 * {@link #verify() Verifies} the signals received by this subscriber, then exposes
+	 * various {@link Assertions assertion methods} on the final state.
+	 * <p>
+	 * Note that like {@link #verify()}, this method will <strong>block</strong> until
+	 * the stream has been terminated (either through {@link Subscriber#onComplete()},
+	 * {@link Subscriber#onError(Throwable)} or {@link Subscription#cancel()}).
+	 * Depending on the declared expectations and actions, notably in case of undersized
+	 * manual requests, such a verification could also block indefinitely. Use
+	 * {@link Duration} {@code duration} parameter to add a timeout on verify()-derived
```

Comment:
I'd rephrase the sentence to "`As a consequence you can use the {@link Duration} {@code duration} parameter to set a timeout.`".